pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -335,6 +335,10 @@ in
 
   blink-cmp-words = super.blink-cmp-words.overrideAttrs {
     dependencies = [ self.blink-cmp ];
+    meta = {
+      description = "Fast, offline word and synonym completion provider for Neovim";
```

```suggestion
      description = "Offline word and synonym completion provider for Neovim";
```

Sorry, realized after pasting the list from the docs, this is probably the more appropriate description.

---

No worries. That makes sense. "Fast" is subjective after all :upside_down_face:. I have made the update